ccache.c

```diff
@@ -2441,6 +2441,31 @@ cc_process_args(struct args *args, struct args **preprocessor_args,
 			free(option);
 			continue;
 		}
+                if (str_startswith(argv[i], "-B")) {
+			char *relpath = make_relative_path(x_strdup(argv[i] + 2));
+			char *option = format("-B%s", relpath);
+			args_add(stripped_args, option);
+			free(relpath);
+			free(option);
+			continue;
+                }
+                if (str_startswith(argv[i], "-iprefix")) {
+			char *relpath = make_relative_path(x_strdup(argv[i+1]));
```

Comment:
> I RTFM'ed GCC and clearly there is a space between iprefix and the directory as mentioned here 

Yes, but GCC is more forgiving than what's documented, so ccache needs to be as well.

> Also, the link you've provided is broken.

Bummer. Here's the link without the extraneous ".": http://www.mail-archive.com/ccache@lists.samba.org/msg01235.html

> Can you please paste the compiler invocation flags here ?

Sorry, I don't understand what you're asking for.